### PR TITLE
bats helpers.bash - minor refactoring

### PR DESCRIPTION
**Pull request by edsantiago**
Almost trivial, but this will greatly simplify my podman-remote
testing of buildah-bats.

Signed-off-by: Ed Santiago <santiago@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
/approve
LGTM

File: tests/helpers.bash [hunk 1]
```diff
@@ -60,24 +64,18 @@ function teardown() {
 }
 
 function _prefetch() {
-    # Tell podman to use the same mirror registries as buildah, to
-    # avoid docker.io
-    export REGISTRIES_CONFIG_PATH=${TESTSDIR}/registries.conf
-
```
Comment (TomSweeneyRedHat) [inline tests/helpers.bash:66]: I thought we somehow referenced this in Podman, but I'm not finding it....

Comment (edsantiago) [reply tests/helpers.bash:66]: Podman uses it - but we no longer use podman for pulls (See #3099). I should've removed these lines as part of that PR, but didn't notice it at the time. This is just belated cleanup.

And, in any event, the new `podman()` helper function now explicitly includes `$REGISTRY_OPTS` as a command-line argument, which is better than an envariable.

**Pull request merged**